src/procedures/proc_pagerank.c

```
@@ -117,14 +117,14 @@ ProcedureResult Proc_PagerankInvoke
 		s = GraphContext_GetSchema(gc, relation, SCHEMA_EDGE);
 		// unknown relation, quickly return
 		if(!s) return PROCEDURE_OK;
-		Delta_Matrix_export(&r, Graph_GetRelationMatrix(g, s->id, false));
+		Delta_Matrix_export(&r, Graph_GetRelationMatrix(g, s->id));
 
 		// convert the values to true
 		info = GrB_Matrix_apply(r, NULL, NULL, GxB_ONE_BOOL, r, GrB_DESC_R);
 		ASSERT(info == GrB_SUCCESS);
```

cppcheck

[error] Null pointer dereference

(nullPointer)